clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -1128,9 +1128,16 @@ void SYCLToolChain::AddSYCLIncludeArgs(const clang::driver::Driver &Driver,
   llvm::sys::path::append(P, "include");
   SmallString<128> SYCLP(P);
   llvm::sys::path::append(SYCLP, "sycl");
+  // This is used to provide our wrappers around STL headers that provide
```

Comment:
Couple of style suggestions:
1. Update the comment at line 1125.
2. Please, re-arrange the code:
from 
```c++
// define path1
// define path2
// set path1 via CC1Args
// set path2 via CC1Args
```
to 
```c++
// define path1
// set path1 via CC1Args
// define path2
// set path2 via CC1Args
```

Reply:
After reading the code more carefully, I realized that suggestion 2 will require code duplication as we define all paths before we can set the first path due to order restrictions. If so, I suggest ignoring my second suggestion.